src/EA.Weee.Web/Areas/Aatf/ViewModels/SubmittedDatesFilterViewModel.cs

```diff
@@ -20,7 +14,7 @@ public class SubmittedDatesFilterViewModel
 
         [Display(Name = "Submitted date")]
         [DataType(DataType.Date)]
-        [EvidenceNoteEndDate(nameof(StartDate))]
+        [EvidenceNoteEndDate(nameof(StartDate), false)]
```

Comment:
Think the SubmittedDatesFilterViewModel_EndDate_ShouldHaveStartDateAttribute unit test needs to be updated to check for false

Reply:
okay

Reply:
Very good point, I was testing the wrong model